config/functions.cfg

```diff
@@ -211,18 +211,18 @@ function git_clone {
   #If OVERRIDE_CONFIGVER is set proceed to clone from that tag
   [[ ! -z "$OVERRIDE_CONFIGVER" ]] && CONFIGVER=$OVERRIDE_CONFIGVER
 
-  echo -e "${GREEN}Cloning into ${CYAN}elrond-config-$ENVIRONMENT${GREEN} with tag ${CYAN}$CONFIGVER${GREEN}...${NC}"
+  echo -e "${GREEN}Cloning into ${CYAN}mx-chain-$ENVIRONMENT-config${GREEN} with tag ${CYAN}$CONFIGVER${GREEN}...${NC}"
   echo -e
-  cd $GOPATH/src/github.com/ElrondNetwork && git clone https://github.com/ElrondNetwork/elrond-config-"$ENVIRONMENT" && cd elrond-config-"$ENVIRONMENT" && git checkout --force $CONFIGVER
+  cd $GOPATH/src/github.com/MultiversX && git clone https://github.com/MultiversX/mx-chain-"$ENVIRONMENT"-config && cd mx-chain-"$ENVIRONMENT"-config && git checkout --force $CONFIGVER
```

Comment:
might use in the entire PR `github.com/multiversx/` instead of `github.com/MultiversX/`

Reply:
Solved ! Great catch !